diff --git a/node_modules/@fos/shared/dist/services/api/auth/types.d.ts b/node_modules/@fos/shared/dist/services/api/auth/types.d.ts
index 66c152c..d497241 100644
--- a/node_modules/@fos/shared/dist/services/api/auth/types.d.ts
+++ b/node_modules/@fos/shared/dist/services/api/auth/types.d.ts
@@ -13,5 +13,5 @@ export interface AccountRegistrationRequest {
     registrationUuid: string;
 }
 export interface OtpVerificationResponse {
-    status: string;
+    uuid: string;
 }
